crypto/nrf_cc310_mbedcrypto/CHANGELOG.rst

```diff
@@ -9,6 +9,43 @@ Changelog - nrf_cc3xx_mbedcrypto
 
 All notable changes to this project are documented in this file.
 
+nrf_cc3xx_mbedcrypto - 0.9.16
+*****************************
+
+New version of the runtime library with the following bug fixes and improvements:
+
+* Built with MBEDTLS_PSA_CRYPTO_KEY_ID_ENCODES_OWNER enabled to ensure support with and without TF-M.
```

Comment:
Ensure what support?

Was nrf_cc3xx_mbedcrypto not supported to be used with TF-M before?

Reply:
Fixed